godel_surface_detection/src/nodes/robot_auto_blending.cpp

```diff
@@ -13,87 +11,112 @@
 #include <fstream>
 #include <iostream>
 
+//Topic
 const static std::string SURFACE_DETECTION_SERVICE = "surface_detection";
 const static std::string SELECT_SURFACE_SERVICE = "select_surface";
 const static std::string PROCESS_PATH_SERVICE = "process_path";
-const std::string GET_AVAILABLE_MOTION_PLANS_SERVICE = "get_available_motion_plans";
+const static std::string GET_AVAILABLE_MOTION_PLANS_SERVICE = "get_available_motion_plans";
 const static std::string SELECT_MOTION_PLAN_SERVICE = "select_motion_plan";
 const static std::string SURFACE_BLENDING_PARAMETERS = "surface_blending_parameters";
+const static int MAX_SERVICECALL_FAILURE = 20;
 
 int main(int argc, char** argv)
 {
   ros::init(argc, argv, "robot_auto_blending");
   ros::NodeHandle nh;
   ros::Rate loop_rate(10);
 
-  std::string ns = ros::this_node::getNamespace();
+  godel_msgs::SurfaceBlendingParameters param_srv;
+  godel_msgs::SurfaceDetection surfacedetection_srv;
+  godel_msgs::SelectSurface::Request req;
+  godel_msgs::SelectSurface::Response res;
+  godel_msgs::ProcessPlanning process_plan;
+  godel_msgs::GetAvailableMotionPlans motionQuery_srv;
+  godel_msgs::SelectMotionPlan motion_srv;
+  std::vector < std::string > plan_names;
+  int serviceCall_failure = 0;
```

Comment:
I submitted another 3 commit around noon yesterday, let me know if there's anything need to update.